packages/ui/src/utils/getErrorMessageReservedFunds.tsx

```diff
@@ -0,0 +1,30 @@
+export const wikiLinkReservedFunds =
+  'https://github.com/ChainSafe/Multix/wiki/Why-are-funds-reserved%3F'
+
+export const getErrorMessageReservedFunds = (
+  identifier: string,
+  requiredBalanceString?: string,
+  reservedString?: string
+) => {
+  if (!requiredBalanceString) return ''
+
+  const reservedMessage = reservedString ? (
+    <span>
+      Note that it includes ${reservedString} that will be reserved.{' '}
+      <a
+        href={wikiLinkReservedFunds}
+        about="_blank"
```

Comment:
It's also probably intended to be target="_blank", although it's a good Berlin club that was on your mind 😅

Reply:
There is a good practice to add rel="noopener noreferrer" for security reason

Reply:
totally!